test/assets/custom-certs/csr-no-san.conf.template

```diff
@@ -0,0 +1,19 @@
+[ req ]
+default_bits = 2048
+prompt = no
+default_md = sha256
+distinguished_name = dn
+
+[ dn ]
+C = il
+ST = il
+L = raanana
+O = redhat
+OU = aaa
```

Comment:
OU = Opragnizational Unit

Can we use something more meaningful here, i.e. `eng`?